corehq/apps/app_manager/models.py

```diff
@@ -329,12 +334,17 @@ def all_property_names(self):
         return names
 
 
+class ParentIndex(DocumentSchema):
```

Comment:
I agree with all of that. I should have gone with `action.indices` from the beginning. :expressionless:  Oh well.